.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "third_party/SEAL"]
 	path = third_party/SEAL
 	url = https://github.com/Microsoft/SEAL
+[submodule "third_party/googletest"]
+	path = third_party/googletest
+	url = https://github.com/google/googletest
```

Comment:
I feel like the googletest should be something separate from the core requirement for the library to be built, but I don't know how we can separate that.